print/components/scheduleEntry/contentNode/ColumnLayout.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <table class="tw-w-full">
-    <tr>
-      <column-layout-column
-        v-for="column in contentNode.data.columns"
-        :key="column.slot"
-        :content-node="contentNode"
-        :column-slot="column.slot"
-      />
-    </tr>
-  </table>
+  <div class="tw-flex tw-flex-wrap">
```

Comment:
Yes I did try and it works quite good in LayoutNG. But after working on this PR, I'm questioning the ColumnLayout in print in general. In the cases of not so much content, I would prefer a layout more like we had in eCamp v2. If you look at Page 22-24 in [Pfingstlager-2023-GRGR (35).pdf](https://github.com/ecamp/ecamp3/files/11782665/Pfingstlager-2023-GRGR.35.pdf) you can see that it would be beneficial to move the side column to the top or bottom.

Reply:
If it now turns out that having the storyboard side-by-side with other content nodes leads to wasted space, maybe the way forward would be to adjust the camp templates so that the storyboard is full-width by default, and the other content nodes are in column layouts above and below it? E.g.
```
+------------------------------------+
|           |                        |
|   Story   |    Security concept    |
|           |                        |
+------------------------------------+
|                                    |
|                                    |
|                                    |
|                                    |
|                                    |
|                                    |
|             Storyboard             |
|                                    |
|                                    |
|                                    |
|                                    |
|                                    |
|                                    |
+------------------------------------+
|                    |               |
|                    |               |
|                    |               |
|      Material      |     Notes     |
|                    |               |
|                    |               |
|                    |               |
+------------------------------------+
```
(Not sure if you were already proposing that)

Reply:
Lets discuss at the hackaton.

Reply:
Camp template layouts were adjusted in #3553. So in future camps, situations like Page 22-24 will only happen if the users explicitly manually set up their layouts this way.